src/Microsoft.SqlTools.ServiceLayer/AutoParameterizaition/SqlParameterizer.cs

```diff
@@ -0,0 +1,128 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.SqlServer.TransactSql.ScriptDom;
+using System.Linq;
+using Microsoft.SqlTools.ServiceLayer.AutoParameterizaition.Exceptions;
+using System.Data.Common;
+using Microsoft.SqlTools.ServiceLayer.Workspace.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.AutoParameterizaition
+{
+    public static class SqlParameterizer
+    {
+        private const int maxStringLength = 300000; // Approximately 600 Kb
+        private static readonly IList<ScriptFileMarker> EmptyCodeSenseItemList = Enumerable.Empty<ScriptFileMarker>().ToList();
+
+        public static SqlScriptGenerator GetScriptGenerator()
```

Comment:
OoC, does this codebase allow inline definitions yet?

`public static SqlScriptGenerator GetScriptGenerator() => new Sql150ScriptGenerator();`

Reply:
Let's try it and see! Converted to expression body definitions.